install/kubernetes/helm/istio/values.yaml

```diff
@@ -180,6 +180,9 @@ global:
     stats:
       prometheusPort: 15090
 
+    # Specify which tracer to use. One of: lightstep, zipkin
+    tracer: ""
```

Comment:
As mentioned above, think this should have a value, which should be "zipkin" as the default.